src/client/pps/pps.proto

```diff
@@ -342,6 +360,15 @@ message RestartDatumRequest {
   repeated string data_filters = 2;
 }
 
+message InspectDatumRequest {
+  string datum_id = 1 [(gogoproto.customname) = "DatumID"];
+  string job_id = 2 [(gogoproto.customname) = "JobID"];
+}
```

Comment:
There's an existing `datum` message, so I can use that, but it has a `hash` which I don't think means the same thing as the hash that's generated by the hash of all files. I think the existing `datum` message is used in the logging codepath. I'll readup and see what makes sense to resolve.